01-Git_Bash_Make/bash_script

```diff
@@ -3,3 +3,4 @@
 mkdir guess_a_number
 cp main.c guess_a_number/
 tar -czvf guess_a_number.tar.gz guess_a_number/
+mkdir guess_a_number/release
```

Comment:
IMO there's some mismatch with the task regarding what is `project dir` (I assume that it's `01-Git_Bash_Make/`).